localstack/utils/archives.py

```diff
@@ -72,3 +72,23 @@ def untar(path: str, target_dir: str):
     mode = "r:gz" if path.endswith("gz") else "r"
     with tarfile.open(path, mode) as tar:
         tar.extractall(path=target_dir)
+
+
+def create_zip_file_cli(source_path, base_dir, zip_file):
+    # Using the native zip command can be an order of magnitude faster on Travis-CI
+    source = "*" if source_path == base_dir else os.path.basename(source_path)
+    command = "cd %s; zip -r %s %s" % (base_dir, zip_file, source)
+    run(command)
```

Comment:
nit: While we're at it, we could change this logic to the new format of `run(..)` (command as list, avoiding shell expansion): 🙏 
```suggestion
    run(["zip", "-r", zip_file, source], cwd=base_dir)
```

Reply:
This improved version seems to fail for some tests. I will merge as it is and look at it after the CLI changes are in.